tests/test-core/utils/js/ConditionalTestUtils.js

```diff
@@ -100,13 +100,44 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     fluid.test.conditionalTestUtils.isBrowserOnWindows = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Windows", "{fluid.browser.platformName}");
+        return fluid.test.conditionalTestUtils.contextValueContains("Win", "{fluid.browser.platformName}");
```

Comment:
I added a number of examples for each platform and a link to a comprehensive Stack Overflow thread. The standard desktop/laptop Windows platform identifier is "Win32" (including on 64-bit architectures).

I also included somewhat shouty warnings about not relying on any of this detection outside of (relatively) controlled testing contexts when routing tests around browser/platform bugs. :)